### DISCUSSION TITLE
Align collapse arrow with title when reducing padding of title

### DISCUSSION THREAD

**Discussion by dsoverby1986** [saadeghi/daisyui#2306] [Q&A]
I wanted to make the collapse title area not so thick, so I reduced the vertical padding, but now the collapse arrow doesn't line up properly with the title text and it looks like whatever padding may be around the arrow is keeping the title area thick. 

Any advice on how to get the arrow to align with the title text?

![image](https://github.com/saadeghi/daisyui/assets/7773023/bda5d26b-d118-4087-99e2-6c0a69e3e5c1)

**Comment by saadeghi** [comment]
Share your code please.  
You can use a [Tailwind Play](https://daisyui.com/tailwindplay/) page.
